gulpfile.js

```diff
@@ -141,10 +145,16 @@ gulp.task('build:intros', ['clean'], function() {
   return Promise.all(recipes.map(function(recipe) {
     return fsReadFile(recipe.slug + '/README.md', 'utf8')
     .then(function(readme) {
-      return renderFile('./src/tpl/intro.html', { markdown: marked(readme) });
+      return renderFile('./src/tpl/intro.html', {
+        markdown: marked(readme)
+      });
     })
     .then(function(content) {
-      return template.writeFile('./dist/' + recipe.slug + '.html', content, { currentRecipe: recipe });
+      return template.writeFile(
```

Comment:
It closed at 78 on the first one, but not on the next one, so left the next one as is.